README.md

```diff
@@ -91,8 +72,9 @@ The full text of the licence is included in [LICENSE.md](https://raw.githubuserc
 
 ## Auxiliary Files
 
-Auxiliary files, such as pre-trained model weights downloaded from the TIA Lab webpage (https://warwick.ac.uk/fac/sci/dcs/research/tia/tiatoolbox), are provided under the [Creative Commons Attribution-NonCommercial-ShareAlike Version 4 (CC BY-NC-SA 4.0) license](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+Auxiliary files, such as pre-trained model weights downloaded from the TIA Centre webpage (https://warwick.ac.uk/fac/sci/dcs/research/tia/tiatoolbox), are provided under the [Creative Commons Attribution-NonCommercial-ShareAlike Version 4 (CC BY-NC-SA 4.0) license](https://creativecommons.org/licenses/by-nc-sa/4.0/).
```

Comment:
This link "https://warwick.ac.uk/fac/sci/dcs/research/tia/tiatoolbox" is also broken. For now we can specify "https://warwick.ac.uk/fac/cross_fac/tia/" and later add the exact location once we have uploaded the models and the webpage is ready.

Reply:
OK